Adaptors/MongoDB/tests/ArmoniK.Core.Adapters.MongoDB.Tests.csproj

```diff
@@ -5,14 +5,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Configuration" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
-    <PackageReference Include="Mongo2Go" Version="3.1.3" />
```

Comment:
Do the tests still run if we remove Mongo2go?

Reply:
Yes they are included in Common.Test